tools/cythonize.py

```diff
@@ -100,6 +100,24 @@ def process_tempita_pyx(fromfile, tofile):
         f.write(pyxcontent)
     process_pyx(pyxfile, tofile)
 
+
+def process_tempita_pxi(fromfile, tofile):
+    try:
+        try:
+            from Cython import Tempita as tempita
```

Comment:
If we shouldn't rely on it, we shouldn't use it. I believe you encountered a problem on some platform?  But I suppose there is nothing lost be at least trying Cython, tempita itself hasn't been updated in a long time and still uses 2to3.